content/en/blog/2025/go-2025-goals.md

```diff
@@ -0,0 +1,136 @@
+---
+title: OpenTelemetry Go 2025 Goals
+linkTitle: Go Goals 2025
```

Comment:
Why the inconsistency in the work order. How about:

```suggestion
title: OpenTelemetry Go 2025 Goals
linkTitle: Go 2025 Goals
```

Reply:
https://github.com/open-telemetry/opentelemetry.io/pull/5966/commits/08720842682f744393c5b0440f50a0923c3f108c